src/common/interfaces/environment-variables.interface.ts

```diff
@@ -10,4 +10,8 @@ export interface IEnvironmentVariables {
   AWS_S3_ENDPOINT: string;
   AWS_S3_BUCKET_NAME: string;
   AWS_S3_PRESIGN_URL_EXPIRY_IN_MINUTES: number;
+  MAIL_HOST: string;
+  MAIL_PORT: number;
+  MAIL_USER: string;
+  MAIL_PASSWORD: string;
```

Comment:
Could you add these to the .env.example as well?

Reply:
Okay bhaiya, will do that.